src/js/services/forms.js

```diff
@@ -98,7 +98,7 @@ export default App.extend({
       channel.request('send', 'fetch:form:data', {
         definition,
         formData: fields.data.attributes || {},
-        formSubmission: response && response.data || {},
+        formSubmission: response && response.data.attributes.response.data || {},
```

Comment:
If there's not a form response does this endpoint return nothing?  If it returns `{}` this'd break.

Reply:
I believe it returns an empty `204` response.

But it should probably be built to not break either way.

Reply:
I think this'd work if it's a 204.  Not sure there's a possible other use case